app/poc-common/form-display/directives/formField.js

```diff
@@ -85,15 +95,25 @@ angular.module('poc.common.formdisplay')
         
         $scope.isTrueFalseQuestion = function (question) {
             var found = _.find(question, function (answer) {
+               // console.log(JSON.parse(JSON.stringify(answer)));
+
                 return answer.uuid === "e1d81b62-1d5f-11e0-b929-000c29ad1d07" || 
                         answer.uuid === "e1d81c70-1d5f-11e0-b929-000c29ad1d07";
             });
             return typeof found !== "undefined";
         };
 
+        $scope.getConceptInAnswers = function (answres, conceptUuid) {
+            return _.find(answres, function (answer) {
+                return answer.uuid === conceptUuid;
+                 //console.log(answer);
```

Comment:
@psbrandt are you sure this needs to go? It's being used in `app/poc-common/form-display/views/formField.html`